[2023-10-17T11:19:05.267+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-28T00:00:00+00:00 [queued]>
[2023-10-17T11:19:05.274+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-28T00:00:00+00:00 [queued]>
[2023-10-17T11:19:05.274+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:05.285+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-02-28 00:00:00+00:00
[2023-10-17T11:19:05.291+0000] {standard_task_runner.py:57} INFO - Started process 541 to run task
[2023-10-17T11:19:05.294+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-02-28T00:00:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpo01htmkm']
[2023-10-17T11:19:05.295+0000] {standard_task_runner.py:85} INFO - Job 63: Subtask waiting_for_data
[2023-10-17T11:19:05.329+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-28T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:05.396+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-28T00:00:00+00:00'
[2023-10-17T11:19:05.404+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:05.404+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:05.404+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:05.404+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:05.411+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210228T000000, start_date=20231017T111905, end_date=20231017T111905
[2023-10-17T11:19:05.425+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:05.437+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:35.105+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-28T00:00:00+00:00 [queued]>
[2023-10-18T10:42:35.113+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-28T00:00:00+00:00 [queued]>
[2023-10-18T10:42:35.113+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:35.128+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-02-28 00:00:00+00:00
[2023-10-18T10:42:35.135+0000] {standard_task_runner.py:57} INFO - Started process 1507 to run task
[2023-10-18T10:42:35.138+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-02-28T00:00:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpaq7aa9nd']
[2023-10-18T10:42:35.139+0000] {standard_task_runner.py:85} INFO - Job 62: Subtask waiting_for_data
[2023-10-18T10:42:35.187+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-28T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:35.272+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-28T00:00:00+00:00'
[2023-10-18T10:42:35.290+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:35.293+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210228T000000, start_date=20231018T104235, end_date=20231018T104235
[2023-10-18T10:42:35.305+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 62 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1507)
[2023-10-18T10:42:35.351+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:35.381+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:04.587+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-28T00:00:00+00:00 [queued]>
[2023-10-18T11:17:04.597+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-28T00:00:00+00:00 [queued]>
[2023-10-18T11:17:04.597+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:04.612+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-02-28 00:00:00+00:00
[2023-10-18T11:17:04.618+0000] {standard_task_runner.py:57} INFO - Started process 281 to run task
[2023-10-18T11:17:04.621+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-02-28T00:00:00+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpd3a0es8g']
[2023-10-18T11:17:04.622+0000] {standard_task_runner.py:85} INFO - Job 79: Subtask waiting_for_data
[2023-10-18T11:17:04.668+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-28T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:04.748+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-28T00:00:00+00:00'
[2023-10-18T11:17:04.759+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:04.761+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:04.761+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:04.762+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:04.773+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210228T000000, start_date=20231018T111704, end_date=20231018T111704
[2023-10-18T11:17:04.795+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:04.830+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:11:41.224+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-28T00:00:00+00:00 [queued]>
[2023-10-19T12:11:41.237+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-28T00:00:00+00:00 [queued]>
[2023-10-19T12:11:41.237+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:11:41.256+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-02-28 00:00:00+00:00
[2023-10-19T12:11:41.267+0000] {standard_task_runner.py:57} INFO - Started process 339 to run task
[2023-10-19T12:11:41.273+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-02-28T00:00:00+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpuclvlf2l']
[2023-10-19T12:11:41.274+0000] {standard_task_runner.py:85} INFO - Job 77: Subtask waiting_for_data
[2023-10-19T12:11:41.335+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-28T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:11:41.432+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-28T00:00:00+00:00'
[2023-10-19T12:11:41.441+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:11:41.442+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:11:41.442+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:11:41.442+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:11:41.454+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210228T000000, start_date=20231019T121141, end_date=20231019T121141
[2023-10-19T12:11:41.483+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:11:41.530+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
